test/commands/summary.spec.js

```diff
@@ -26,7 +26,6 @@ test("summary should execute summary command on fixtures 'result-test1.json'", a
     /\(48\) MIT - \(1\) BSD-3-Clause - \(3\) ISC.*$/,
     /.*/
   ];
-  tape.plan(lines.length * 2);
```

Comment:
This plan seemed correct.

Reply:
Finally, catch the error. The `@nodesecure/scanner` changed a bit and the fixture was simply outdated.